src/blockchain/staking/history.ts

```diff
@@ -0,0 +1,68 @@
+import { Block, Log } from '@ethersproject/abstract-provider'
+import { BaseProvider } from '@ethersproject/providers'
+import { Abi } from '@xtoken/abis'
+import { formatUnits, Interface } from 'ethers/lib/utils'
+
+import { IStakeHistory } from '../../types/xToken'
+
+import { getXtkStakingContract } from './helper'
+
+const STAKING_HISTORY_START_BLOCK = 12838146
```

Comment:
Pretty obvious looking at the file that this would be Staking History start block, so no need to be specific on the constant naming until and unless we're moving this to global constant
```suggestion
const fromBlock = 12838146
```